appengine/helloworld-new-plugins/gradlew.bat

```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
Ditto. License header if feasible, otherwise if this is auto-generated, consider removing and add to .gitignore

Looks like at least one person does add these to .gitignore. http://www.w3ma.com/android-studio-gitignore-avoid-useless-files/